LoopKitUI/Extensions/DateFormatter.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  DateFormatter.swift
+//  LoopKitUI
+//
+//  Created by Anna Quinlan on 7/7/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+
+extension DateFormatter {
```

Comment:
Added because it's needed by glucose chart, which is needed by insulin model chart